crates/pgt_completions/src/complete.rs

```diff
@@ -14,7 +14,7 @@ pub struct CompletionParams<'a> {
     pub position: TextSize,
     pub schema: &'a pgt_schema_cache::SchemaCache,
     pub text: String,
-    pub tree: Option<&'a tree_sitter::Tree>,
```

Comment:
tree sitter cannot fail and with the new pattern this doesnt need to be an option anymore